.gitignore

```
@@ -2,8 +2,7 @@
 # Edit at https://www.toptal.com/developers/gitignore?templates=visualstudiocode,vim,webstorm,macos,linux,windows,zsh,node
 
 .env
-db/schema.sql
-
+# db/schema.sql
```

왜 주석 처리 하셨나요? 다시 풀어주세요.
```suggestion
db/schema.sql
```